relo:
    priorDir: "/home/yixin-f/fast-lio2/src/data_loc"
    cloudTopic: "/cloud_registered"
    poseTopic: "/Odometry"
    searchDis: 5.0
    searchNum: 3  # if the cloud is dense, please reset it to 1
    trustDis: 5.0
    # 0: ICP 1: AA-ICP 2: Our Fast ICP 3: Our Robust ICP 4: Our Fast and Robust ICP 5: ICP Point-to-plane
    # 6: Our Fast and Robust ICP point to plane 7: Sparse ICP 8: Sparse ICP point to plane
    regMode: 4 # (0~8), but methods 5, 6 and 8 need normal information

    extrinsic_T: [1.77, -0.00, -0.05] # lidar to imu # parkinglot
    extrinsic_R: [ -1, 0, 0, 
                    0, -1, 0, 
                    0, 0, 1]

    # extrinsic_T: [ 0.031, -0.10, 0.11]  # seu velo16
    # extrinsic_R: [ 0.9994142, 0.03275957, 0.00990251,
    #               -0.03266505,  0.99942062, -0.00956125,
    #               -0.01021,  0.00923218,  0.99990526]

    # extrinsic_T: [ 0.071, -0.053, 0.097 ] # zhongyifeng
    # extrinsic_R: [ 0.9935, -0.0140, -0.1128,
    #                0.0137, 0.9999, -0.0040,
    #                0.1128, 0.0024, 0.9936]

    # extrinsic_T: [ 0.05512, 0.02226, -0.0297 ]   # mid360
    # extrinsic_R: [ 1, 0, 0,
    #                0, 1, 0,
    #                0, 0, 1]